graviti/utility/user.py

```diff
@@ -393,6 +394,8 @@ def __getitem__(self, key: Union[int, str]) -> _V:
     def __setitem__(self, key: Union[int, str], value: _V) -> None:
```

Comment:
can we add a `setitem` method only accept str key.
then we can use this `setitem` in `__init__` and `__setitem__`?